pkg/jobmanager/start.go

```diff
@@ -92,17 +100,19 @@ func (jm *JobManager) start(ev *api.Event) *api.EventResponse {
 			// * Job completed successfully
 			// * Job was cancelled
 			var eventToEmit event.Name
-			if j.IsCancelled() {
-				log.Infof("Job %+v completed cancellation", j)
-				eventToEmit = EventJobCancelled
-			} else {
```

Comment:
It seems previously we double-emitted `EventJobCancelled`. Added a little-bit more correct logic for `EventJobPaused` and fixed the logic for `EventJobCancelled` for consistency.